.github/workflows/rebase.yml

```diff
@@ -0,0 +1,18 @@
+# https://github.com/marketplace/actions/automatic-rebase  (https://github.com/cirrus-actions/rebase)
+on:
+  issue_comment:
+    types: [created]
+name: Automatic Rebase
+jobs:
+  rebase:
+    name: Rebase
+    if: github.event.issue.pull_request != '' && contains(github.event.comment.body, '/rebase') && github.event.comment.author_association == 'MEMBER'
```

Comment:
I'm guessing comments on issues would cause this to be false: `github.event.issue.pull_request != ''` Where comments on PRs cause it to be true. I've seen a couple of other ways to do this, but I'm not sure what is ideal.

Also, I'm curious if we should use a different (more intentional) trigger word; I can see typing "merge/rebase" on a PR and triggering this by accident. :sweat_smile: 

Maybe we could use something like: `contains(github.event.comment.body, '/trigger_rebase')`? I'm curious what others think!

Reply:
I'm +1 with following dependabot's simplicity. Copy pasting from one of dependabot's PRs:

```text
@dependabot rebase will rebase this PR
@dependabot recreate will recreate this PR, overwriting any edits that have been made to it
@dependabot merge will merge this PR after your CI passes on it
@dependabot squash and merge will squash and merge this PR after your CI passes on it
@dependabot cancel merge will cancel a previously requested merge and block automerging
```

this is how you can interact with the bot that manages our dependencies. I would use:

```text
/rebase -> will rebase this PR on top of master
```

Reply:
@jacobherrington  and @rhymes . Thanks very much for the review. I don't know if there are any changes or actions I should be taking to get this merged.

Reply:
@rhymes To clarify, I think the way it's currently written if I wrote a comment like this:

```
I think we might want to merge/rebase this PR...
```

I suspect the branch would be rebased because the above string contains `/rebase`. I was wondering if we should make it a more specific command so that we don't accidentally rebase PRs.

https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#contains

@awasum I'm not sure either :) I want to give other folks on our team a chance to weigh-in on this before I approve though. It looks good otherwise! :+1:

Reply:
oh got it! should we use `startsWith`? we definitely don't want to rebase if any of us writes `/rebase` in the middle of a sentence :D

Reply:
I think `startsWith` would be a great idea!

@awasum do you mind using the `startsWith` function instead?

https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#startswith

Reply:
I don't mind. I will take a look and push a new commit soon

Reply:
@rhymes and @jacobherrington . I have added a new commit to use startsWith() instead of contains: https://github.com/thepracticaldev/dev.to/pull/9018/commits/30af093af765b76659bf67c50a2eaf19964f30b9

I am resolving this conversation as completed.